### DISCUSSION TITLE
[Enhancement] Add permalink(s) to download extension for users convenience in account settings.

### DISCUSSION THREAD

**Discussion by akc3n** [simple-login/app#1931] [Feature Requests]
# Purpose

To enhance convenience and efficiency for the end user experience by facilitating a list of extensions to download based on the user's browser preference.
## Current

Scenario: A new user registers for your service using the '[Create Account](https://app.simplelogin.io/auth/register)' link, or optionally, can choose to '[Sign up with Proton](https://proton.me/support/create-simplelogin-account-proton-account)'. Curiously exploring their dashboard settings of their new account, they come across the section featuring '[Include website address in one-click alias creation on browser extension](https://app.simplelogin.io/dashboard/setting#include_website_in_one_click_alias)'. 

Dashboard Settings: 

![div_id-include_website_in_one_click_alias](https://github.com/simple-login/app/assets/71854332/28b561d0-913c-4b68-bfde-f9e5d0269843)

However, and for example, in order to install this extension, they now face the manual task of either having to navigating through your web pages to find the link, or opening a new tab to type out "_simplelogin browser extension_", or selecting keywords from the section heading, right-clicking, and choosing to search '_x-searchengine_ for one-click alias creation on browser extension'.
Afterwards. Clicking on the search results page listing 'SimpleLogin:Receive & Send emails anonymously' that's specific to their browser and to open the Chrome Web Store page, another click when selecting 'Add to Chrome', and _one more_ **final** click to 'Add extension' from the browsers internal pop prompt.  
## Proposal

I've created a basic example using Google Chrome from a desktop browser perspective, following the branding guidelines - [Using the Chrome Web Store badge](https://developer.chrome.com/docs/webstore/branding/#using-the-chrome-web-store-badge)

![div_id-include_website_in_one_click_alias](https://github.com/simple-login/app/assets/71854332/5a07a202-79b6-409e-b655-698cb95ecb2a)

If the browser of choice for the end user is Chrome, they can now easily click on the "Available in the Chrome Web Store" badge and will be directed to [https://chrome.google.com/webstore/detail/simpleloginreceive-send-e/dphilobhebphkdjbpfohgikllaljmgbn](https://chrome.google.com/webstore/detail/simpleloginreceive-send-e/dphilobhebphkdjbpfohgikllaljmgbn)

Additional browser extension manager badges would be included and provide a direct link to the extension for download.
# Consclusion

The SimpleLogin app already provides a [GIF](https://simplelogin.io/images/one-click-alias.gif), that is thoughtfully created to illustrate the simplicity and convenience this extension can offer in the users dashboard settings. 

So, I am requesting a feature enhancement discussion and suggesting to [add the code in this section here](https://github.com/simple-login/app/blob/eb8f8caeb893d9bb1665d988bc4cdedc1c6ed8a9/templates/dashboard/setting.html#L513-L548).

This will be helpful to the end user by including the links to various browser's extension managers hosting the SimpleLogin browser extension.

Source: [Chrome for Developers > Documentation > Chrome Web Store > Publish your extension > Branding Guidelines](https://developer.chrome.com/docs/webstore/branding/)

**Comment by ghost** [comment]
I don’t agree with you.

Google is evil corporation. They log literally EVERYTHING that you do on their servers. After that they STOLE all your data and SELL or just LEAK ([proof](https://firewalltimes.com/google-data-breach-timeline/)) all collected data.

SilmpleLogin is purchased by Proton - the main company that fights against surveillance and for E2EE. So publishing the banner from evil corporation is not a good idea.

Also they have such links (bottom), but without banner:
[Screenshot](https://github.com/simple-login/app/assets/149334202/b7e56106-343e-47f8-9bae-bb35cd2aa77e)
